transport-protocol/src/custom.rs

```diff
@@ -254,6 +254,14 @@ pub fn parse_request_proxy_psa_attestation_token(
     proto.get_challenge().to_vec()
 }
 
+pub fn serialize_sgx_collateral(collateral: &transport_protocol::SgxCollateral) -> TransportProtocolResult {
```

Comment:
Missing `rustdoc` comments on these public functions.  Also, these small single-liners can be marked as `inline`.